kmod/core/core.c

```diff
@@ -48,6 +48,10 @@
 #include <asm/cacheflush.h>
 #include "kpatch.h"
 
+#ifdef DISTRO_UBUNTU
+#include <generated/utsrelease.h>
+#endif
```

Comment:
I think DISTRO_UBUNTU isn't needed because generated/utsrelease.h also exists for upstream kernels so it can be included unconditionally.